crates/rolldown_plugin_reporter/src/lib.rs

```diff
@@ -316,6 +319,21 @@ impl Plugin for ReporterPlugin {
           utils::log_info(&info);
         }
       }
+    } else if self.warn_large_chunks {
```

Comment:
Curious about https://github.com/vitejs/rolldown-vite/blob/7ea4f545/packages/vite/src/node/plugins/reporter.ts#L265-L269 use `else`, but we use `else if`